src/views/FlowNotify.vue

```diff
@@ -18,30 +18,41 @@ export default {
 	name: 'FlowNotify',
 	components: {},
 	props: {
+		modelValue: {
+			default: '',
+			type: String,
+		},
 		value: {
-			default: JSON.stringify({ inscription: '' }),
+			default: '',
 			type: String,
 		},
 	},
+	emits: ['input', 'update:model-value'],
 	data() {
+		const inscription = (this.modelValue || this.value)
+			? JSON.parse(this.modelValue || this.value).inscription
+			: ''
 		return {
-			inscription: '',
+			inscription,
 			placeholder: t('flow_notifications', 'Choose a notification title (optional)'),
 		}
 	},
-	computed: {
-		currentInscription() {
-			if (!this.value) {
-				return ''
-			}
-			return JSON.parse(this.value).inscription
+	watch: {
+		modelValue() {
+			this.inscription = JSON.parse(this.modelValue || this.value).inscription
+		},
+		value() {
+			this.inscription = JSON.parse(this.modelValue || this.value).inscription
 		},
 	},
 	methods: {
 		emitInput(value) {
-			if (value !== null) {
-				this.$emit('input', JSON.stringify({ inscription: value.target.value }))
+			if (value === null) {
+				return
 			}
+			this.inscription = value.target.value
```

Comment:
Yes... this should not be necessary as `v-model` should take care of it. Maybe the `emitInput` function is triggered before the `v-model` sync.
I'd suggest moving this to a watch on `inscription` then you can be sure the value was already updated. This would also save you the check for `value === null` (where I'm also unsure if it's needed).

Reply:
Yeah, totally right. Interesting too see that vue let's you solve things in different ways. This one wasn't suggested to me before :sweat: but I followed and replaced this event handler with the watcher.